scripts/init.sh

```diff
@@ -9,7 +9,7 @@ if [[ "$docker_version" == 2*.* ]]; then
   HOST='' \
   PORT='' \
   PROJECT='' \
-  docker compose run server python mage_ai/cli/main.py init $PROJECT_NAME
+  docker-compose run server python mage_ai/cli/main.py init $PROJECT_NAME
```

Comment:
could you not update this script?

Reply:
This is bug for running becuase of docker compose running cli with docker-compose